install-and-configure/advanced-configuration/eks-optimized/aws-amp-integration.md

```diff
@@ -0,0 +1,259 @@
+# Amazon Managed Service for Prometheus (AMP)
+
+## Overview
+
+There are three methods to use Kubecost with AMP:
+
+* [AWS Agentless AMP](/install-and-configure/advanced-configuration/eks-optimized/kubecost-agentless-amp.md)
+* [AWS Distro for Open Telemetry](/install-and-configure/advanced-configuration/eks-optimized/kubecost-aws-distro-open-telemetry.md)
+* [AMP with remote_write](/install-and-configure/advanced-configuration/eks-optimized/amp-with-remote-write.md)
+
+The below guide provide a high-level overview of using AMP with Kubecost. The links above provide detailed instructions for each method.
+
+{% hint style="info" %}
+Note: Using AMP allows multi-cluster Kubecost with EKS-Optimized licenses
+{% endhint %}
```

Comment:
Can I get some context to what this means?

Reply:
AMP is the only method for multi-cluster eks-optimized to work.

Reply:
federated etl is enterprise only